examples/kubectl.rs

```diff
@@ -163,14 +163,21 @@ impl App {
             std::fs::read_to_string(&pth).with_context(|| format!("Failed to read {}", pth.display()))?;
         for doc in multidoc_deserialize(&yaml)? {
             let obj: DynamicObject = serde_yaml::from_value(doc)?;
+            let namespace = {
+                if let Some(ns) = obj.metadata.namespace.clone() {
+                    Some(ns)
+                } else {
+                    self.namespace.clone()
+                }
+            };
```

Comment:
think you can do this shorter with combinators

```suggestion
            let namespace = obj.metadata.namespace.clone().or_else(|| self.namespace.clone();
```